docs/packaging/updating-an-existing-package.md

```diff
@@ -167,6 +167,21 @@ Full release notes:
 - [ ] Package was built and tested against unstable
 ```
 
+### Other commit message format examples
+
+In the cases where you are not updating a package to a new version, but simply applying a patch or fixing an existing version, the following commit message shortlogs are considered suitable:
+
+- Add the `homepage` metadata key:
+  - `package: Add homepage`
+- Fix a packaging issue (including adding a patch to fix the package)
+  - `package: Fix (...)`
+- Updating a README, monitoring.yml file or other stuff that does not directly _change_ the package, meaning the package release number didn't need a bump:
+  - `package: [NFC] Add README.md and monitoring.yml`
+
+`[NFC]` is an abbreviation adopted from the clang compiler suite, which expands to "Non Functional Change".
+
+The overall reasoning for the git shortlog formats shown above is to enable quick and efficient "at-a-glance" parsing of git shortlogs for Solus Staff.
```

Comment:
Can you reprhase this sentence with minimal jargon (git, shortlogs, at-a-glance) ?

Reply:
Reads fine to me. Will need someone more familiar with the style guide to chip in here.

Reply:
If we go with my suggestion above about heading hierarchy, I would like this bit to be in the above "Commit message format" section. Something like "We use a git commit message format to enable quick and efficient parsing of git commit messages."

This does a couple of things:

1. Sets the reasoning up front
2. Immediately introduces the idea of readable commit messages
3. Reduces jargon; the reader doesn't need to know what a shortlog is
4. Makes the language more inclusive; Solus staff aren't the only people that might want to read commit messages